src/order.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Order(props) {
+  return (
+    <div className="order">
+      <img src={`/food-${props.foodId}.jpg`}/>
+      <img src={`/drink-${props.drinkId}.jpg`}/>
+      <img src={`/side-${props.sideId}.jpg`}/>
```

Comment:
not destructuring props is a bit of a code smell to me, but I also recognize that is maybe just me.